classification/notebooks/24_exploring_hyperparameters_on_azureml.ipynb

```diff
@@ -984,12 +904,27 @@
     "saved_model.predict(image)\n",
     "```"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 7. Clean up\n",
+    "\n",
+    "To avoid unnecessary expenses, all resources which were created in this notebook need to get deleted once parameter search is concluded. To simplify this clean-up step, we recommend creating a new resource group to run this notebook. This resource group can then be deleted, e.g. using the Azure Portal, which will remove all created resources."
```

Comment:
Should this be as a note at the begining of the notebook? At this point the user has probably already run through the notebook steps

Reply:
Good point. Added note in the beginning.